Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.17 as builder
```

Comment:
Could we separate Go upgrade in a separate PR?

Reply:
Sure, opened here: https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/867